op-challenger/game/fault/types/position_test.go

```diff
@@ -85,7 +85,7 @@ func TestGINConversions(t *testing.T) {
 		pos := NewPosition(test.Depth, test.IndexAtDepth)
 		require.Equal(t, pos, from)
 		to := pos.ToGIndex()
-		require.Equal(t, test.GIndex, to)
+		require.Equal(t, test.GIndex, to.Uint64())
```

Comment:
We need to add cases to each of these unit tests that use positions where the indexAtDepth is greater than the max uint64 value to confirm each individual function still returns the right result.